ci/prow/make_config.go

```diff
@@ -1553,10 +1572,15 @@ func main() {
 		}
 		parseSection(config, "presubmits", generatePresubmit, nil)
 		parseSection(config, "periodics", generatePeriodic, generateGoCoveragePeriodic)
+		generateOtherJobConfigs("periodics", func(repo repositoryData) bool {
+			return !repo.Processed && repo.EnableGoCoverage
```

Comment:
Do you really need .Processed? It seems to me that you're iterating only once over the repos...

Reply:
Yes we need.
In the loop of `parseSection`, we call `generateGoCoveragePeriodic` to process all repos that have both `go-coverage` config in `presubmits` section and config in `periodics` section.
In the loop of `generateOtherJobConfigs`, we call `generateGoCoveragePeriodic` to process all others, that is - all repos that have `go-coverage` config in `presubmits` section but no config in `periodics`.